master/internal/api_runs.go

```diff
@@ -726,6 +729,11 @@ func archiveUnarchiveAction(ctx context.Context, archive bool, runIDs []int32,
 	for _, cand := range runCandidates {
 		visibleIDs.Insert(cand.ID)
 		switch {
+		case cand.ExpArchived:
```

Comment:
I think we want a test for this case in `api_runs_intg_test.go`

Reply:
i'll add test later today.

Reply:
fixed